hugo.toml

```diff
@@ -0,0 +1,344 @@
+baseURL = 'http://knml.edu.pl/'
+languageCode = 'pl-PL'
+title = 'Koło Naukowe Machine Learning Politechniki Rzeszowskiej'
+theme = 'hugo-profile'
+
+Paginate = 3
+enableRobotsTXT = true
+
```

Comment:
Ten plik ma złą nazwę. Zgodnie z dokumentacją powinien nazywać się: `config.toml` (https://gohugo.io/getting-started/quick-start/#configure-the-site). 

Jego brak skutkuje u mnie błędem: 
```
❯ hugo server
Error: Unable to locate config file or config directory. Perhaps you need to create a new site.
       Run `hugo help new` for details.
```